### DISCUSSION TITLE
How do i takedown repository that's basically a replica of a brand

### DISCUSSION THREAD

**Discussion by M33P0** [community/community#30760] [Other Feature Feedback, Questions, & Ideas] [answered]
So our client has requested us to takedown a repository that is a replica of our client's website.
We have tried reporting it through the Trademark abuse form https://support.github.com/contact/trademark-policy

But we have got the below response:
```
Thank you for taking the time to file a trademark policy violation report. Our legal team has reviewed your trademark policy violation report, and based on the facts presented to us, we are unable to determine that a violation of our trademark policy has occurred. We will not be able to take any further action at this time.

I'm very sorry for any disappointment. Please let us know if you have any other questions.
```
We want to take it down as any attacker can clone the repository and host a phishing page to scam users.

What else can we do to bring the repository down?

**Comment by zubair2001** [comment, marked as answer]
maybe try asking the owner of that repo

**Reply by M33P0** [reply]
Thank you, will try this.

**Comment by airtower-luna** [comment]
Find a lawyer who's familiar with trademark law and discuss if you have any other legal options, and if yes whether it'd make sense to pursue them. This is really not something the community can help with.

**Reply by M33P0** [reply]
Thank you, will probably consider it.
